.github/ISSUE_TEMPLATE/Resource_proposal.md

```diff
@@ -0,0 +1,21 @@
+---
+name: New resource proposal
+about: If you have a new resource proposal that you think should be added to this resource module.
+---
+<!--
+    Thank you for contributing and making this resource module better!
```

Comment:
You cannot propose a new resource to this module

Reply:
@johlju - as we can't add new resources to this module, I reckon we should remove this as an option perhaps?